spec/models/manageiq/providers/cloud_manager/configuration.spec.rb

```diff
@@ -0,0 +1,12 @@
+describe ManageIQ::Providers::CloudManager::Provision::Configuration do
+  context 'Userdata payload checks' do
+    it "script is encoded using base64" do
+      template = FactoryGirl.build(:customization_template)
+      template.script = "#cloud-init"
```

Comment:
You can one-line this as `template = FactoryGirl.build(:customization_template, :script => "#cloud-init")`